standard/conversions.md

```diff
@@ -77,7 +78,11 @@ However, dynamic conversions ([§10.2.10](conversions.md#10210-implicit-dynamic-
 
 An identity conversion converts from any type to the same type. One reason this conversion exists is so that a type T or an expression of type T can be said to be convertible to T itself.
 
-Because `object` and `dynamic` are considered equivalent there is an identity conversion between `object` and `dynamic`, and between constructed types that are the same when replacing all occurrences of `dynamic` with `object`.
+In some cases there is an identity conversion between types that are not exactly the same, but are considered equivalent. Such identity conversions exist:
+
+- between `object` and `dynamic`.
+- between tuple types with the same arity, when an identity conversion exists between each pair of corresponding element types.
+- between constructed types that are the same when replacing constituent types with identity convertible ones.
```

Comment:
Possible alternative phrasing:

```suggestion
- between constructed types built from the same generic type where there exists an identity conversion between each corresponding type argument.
```

Reply:
Adopted a variant of your phrasing in upcoming commit